_sass/_layout.scss

```diff
@@ -6,7 +6,6 @@
     min-height: 40px;
     background: $brand-color;
     height: 60px;
-
```

Comment:
I like to have a linebreak before comments, but I guess that's a matter of taste.